temporian/core/event_set_ops.py

```diff
@@ -4436,3 +4437,115 @@ def assign(
         from temporian.core.operators.glue import assign
 
         return assign(self, **others)
+
+    def before(
+        self: EventSetOrNode,
+        timestamp: Union[int, float, datetime],
+    ) -> EventSetOrNode:
+        """Filter events [`EventSet`][temporian.EventSet] that happened before a
+        particual timestamp.
+
+        The timestamp can be a datetime if the EventSet's timestamps are unix
+        timestamps.
+
+        The comparison is strict, meaning that the obtained timestamps would be
+        less than (`<`) the provided timestamp.
```

Comment:
Would it make sense to say something like?:
This operation is equivalent to "input.filter(input.timestamps() < timestamp)"

Reply:
I can add it, do you think I should also add a link to filter?